src/cljs/athens/views/node_page.cljs

```diff
@@ -244,9 +263,11 @@
   [ident]
   (let [{:keys [block/uid node/title] :as node} (db/get-node-document ident)
         editing-uid @(subscribe [:editing/uid])
-        timeline-page? (is-timeline-page uid)]
+        timeline-page? (is-timeline-page uid)
+        show-page-menu? (r/atom false)
+        page-menu-position (r/atom {:x 0 :y 0})]
```

Comment:
Your `atom` skills have leveled 🆙  I would probably merge the atoms into atom. Clojurians typically like to co-locate all stateful (read: bad) things in one place. This is good from a design perspective.

Another reason to keep state together is that it's just one less variable/parameter we have to keep track of. For example, it would not be great if we had an individual atom for each key-value in `block-component`.

Something that I haven't explained properly, and I don't think I fully understand it yet either, is why sometimes we create the atom in `x-el` and sometimes in `x-component`.